rewrite-java-test/src/test/java/org/openrewrite/java/recipes/MissingOptionExampleTest.java

```diff
@@ -0,0 +1,129 @@
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class MissingOptionExampleTest implements RewriteTest {
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new MissingOptionExample())
+          .parser(JavaParser.fromJavaVersion().classpath(JavaParser.runtimeClasspath()));
+    }
+
+    @DocumentExample
+    @Test
+    void lacksExample() {
+        rewriteRun(
+          java(
+            """
+              import org.openrewrite.Option;
+              import org.openrewrite.Recipe;
+
+              class SomeRecipe extends Recipe {
+                  @Option(displayName = "Test", description = "Test")
+                  private String test;
+              
+                  @Override
+                  public String getDisplayName() {
+                      return "Find missing `@Option` `example` values";
+                  }
+                  @Override
+                  public String getDescription() {
+                      return "Find `@Option` annotations that are missing `example` values.";
+                  }
+              }
+              """,
+            """
+              import org.openrewrite.Option;
+              import org.openrewrite.Recipe;
+
+              class SomeRecipe extends Recipe {
+                  /*~~(Missing example value for documentation)~~>*/@Option(displayName = "Test", description = "Test")
+                  private String test;
+              
+                  @Override
+                  public String getDisplayName() {
+                      return "Find missing `@Option` `example` values";
+                  }
+                  @Override
+                  public String getDescription() {
+                      return "Find `@Option` annotations that are missing `example` values.";
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void hasExampleAlready() {
+        rewriteRun(
+          java(
+            """
+              import org.openrewrite.Option;
+              import org.openrewrite.Recipe;
+
+              class SomeRecipe extends Recipe {
+                  @Option(displayName = "Test", description = "Test", example = "true")
+                  private boolean test = true;
+              
+                  @Override
+                  public String getDisplayName() {
+                      return "Find missing `@Option` `example` values";
+                  }
+                  @Override
+                  public String getDescription() {
+                      return "Find `@Option` annotations that are missing `example` values.";
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void skipBoolean() {
```

Comment:
I don't understand the intention of this test

Reply:
Ah, I get it. Booleans don't need ab example.

Reply:
indeed, we already insert [`false` by default for booleans](https://github.com/openrewrite/rewrite-recipe-markdown-generator/blob/821ab474934d72d21a5ec5b956ea92b2c155759f/src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt#L1064-L1065).